### PR TITLE
CCD-1343- Missing MetaData when reaching case-doc-am-api

### PR DESCRIPTION
**Pull request by smathangi**
### Change description ###

Method rename - 263e8fdffa04b492b6945a46b5d95a00a9749bed
bug fix with tests - 15a6432e3e999f6b5b6f35bdd75f1b746a57c744
constants clean up - 522b82c73c043bdbb8dedaa797594f26c15599e8

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java [hunk 1]
```diff
@@ -282,6 +282,25 @@ void shouldBeForbiddenWhenPatchingMetaDataOnDocumentWithIncorrectDocumentHashTok
...
+
+        CaseDocumentsMetadata metadata = CaseDocumentsMetadata.builder()
+            .caseId(CASE_ID_VALUE)
+            .caseTypeId(CASE_TYPE_ID_VALUE)
+            .jurisdictionId(JURISDICTION_ID_VALUE)
+            .documentHashTokens(List.of(new DocumentHashToken(DOCUMENT_ID.toString(), "567890976546789")))
+            .build();
+
+        mockMvc.perform(patch(MAIN_URL + ATTACH_TO_CASE_URL)
+                            .headers(createHttpHeaders("xui_webapp"))
```
Comment (sullivangareth) [inline src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java:296]: Use existing constant `SERVICE_NAME_XUI_WEBAPP`

File: src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java [hunk 2]
```diff
@@ -397,7 +397,8 @@ public void shouldNotAllowPatchDocumentByDocumentIdWhenServiceIsNotAuthorised()
     @Test
     public void shouldNotPatchMetaDataOnDocuments() {
         doThrow(ForbiddenException.class).when(documentManagementService).checkServicePermission(
-            eq(setDocumentMetaData()),
+            eq("BEFTA_CASETYPE_2_1"),
+            eq("BEFTA_JURISDICTION_2"),
```
Comment (sullivangareth) [inline src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java:401]: Use existing `BEFTA_JURISDICTION_2`

Comment (sullivangareth) [reply src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java:401]: This goes for all other instances in this class...

File: src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java (hunk 1 above)
Comment (sullivangareth) [reply src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java:296]: Even better get rid of `SERVICE_NAME_XUI_APP` altogether from this class, and replace with references to static import of

`uk.gov.hmcts.reform.ccd.documentam.apihelper.Constants.XUI_WEBAPP`

File: src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java (hunk 2 above)
Comment (smathangi) [reply src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java]: cleaned up here 522b82c73c043bdbb8dedaa797594f26c15599e8

File: src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java (hunk 1 above)
Comment (smathangi) [reply src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java]: Constants is a main java class. So, removed from there and added to required tests. 
cleaned up here 522b82c73c043bdbb8dedaa797594f26c15599e8

**Review by KleoG — APPROVED** [review]

**Review by sullivangareth — APPROVED** [review]

**Pull request merged**